frontend/jupyter/src/app/pages/form/form-new/form-workspace-volume/form-workspace-volume.component.ts

```diff
@@ -52,6 +52,8 @@ export class FormWorkspaceVolumeComponent implements OnInit, OnDestroy {
     this.volGroup.get('mount').setValue('/home/jovyan');
     this.volGroup.enable();
     this.volGroup.get('newPvc.spec.storageClassName').disable();
+    this.panelOpen = true;
```

Comment:
When testing, this one doesn't seem to work for me for workspace volume, either adding a new volume or adding an existing volume.